public/articles/0/template.html

```diff
@@ -7,17 +7,19 @@
         <title> orgic || wip </title> <!-- replace wip with article title -->
         <link rel="stylesheet" href="../../main.css">
 		<img src="../../logo.png" height="125px" width="125px" class="float-right pagelogo">
-		<div class="pagename">
-			<a href="../../index.html" align="center"><p> ←back to home </p></a>
-		</div>
         <script src="https://cdnjs.cloudflare.com/ajax/libs/mammoth/1.4.19/mammoth.browser.js"></script>
         <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/2.6.1/jszip.js"></script>
         <script src="../../scripts/odt.js"></script>
         
         
     </head>
     <body> <!-- edit the information for these default fields to match the article obviously! -->
-        <div class="article" align="center">
+        <header>
+            <div class="pagename">
```

Comment:
Class can be added directly to `<header>` element to prevent the use of unnecessary `<div>` elements